jkube-kit/parent/pom.xml

```diff
@@ -677,6 +677,14 @@
       </dependency>
       <!-- == test ====================================== -->
 
+      <dependency>
+        <groupId>io.fabric8</groupId>
+        <artifactId>kubernetes-client-api</artifactId>
+        <type>test-jar</type>
+        <version>${version.kubernetes-client}</version>
+        <scope>test</scope>
+      </dependency>
```

Comment:
Considering the Fabric8 BOM is added as a dependencyManagement entry? is this entry necessary?

Reply:
project fails to compile if I remove this entry:
```
 [INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] 'dependencies.dependency.version' for io.fabric8:kubernetes-client-api:test-jar is missing. @ line 133, column 17
[ERROR] 'dependencies.dependency.version' for io.fabric8:kubernetes-client-api:test-jar is missing. @ line 52, column 17
 @ 
[ERROR] The build could not read 2 projects -> [Help 1]
[ERROR]   
[ERROR]   The project org.eclipse.jkube:jkube-kit-common:1.14-SNAPSHOT (/home/rokumar/work/repos/jkube/jkube-kit/common/pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for io.fabric8:kubernetes-client-api:test-jar is missing. @ line 133, column 17
[ERROR]   
[ERROR]   The project org.eclipse.jkube:jkube-kit-resource-helm:1.14-SNAPSHOT (/home/rokumar/work/repos/jkube/jkube-kit/resource/helm/pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for io.fabric8:kubernetes-client-api:test-jar is missing. @ line 52, column 17
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException

```